addon/styles/partials/_uni-select.scss

```diff
@@ -1,3 +1,29 @@
+.uni-select-wrapper--use-alias {
+  position: relative;
+
+  select.uni-select {
+    z-index: 2;
+    opacity: 0;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+
+    &:focus + div.uni-select {
```

Comment:
as we discussed offline, this approach avoids duplicated css, because we want the exact same styles in the "fake select". Also we only want to hide the real select, but make sure it keeps the exact same size so the user doesn't realize anything. (So we really need both of the elements to have the same styles)